clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -29,16 +29,14 @@
 @CommonComponent
 public class ServerStoreImpl implements ServerStore {
 
-  private static final int OFFHEAP_CHAIN_SEGMENTS = 16;
```

Comment:
This change has to do with a completely unrelated experiment of mine I accidentally committed together with the stats work. It should be reverted.